tests/check_all_tables.pl

```diff
@@ -37,7 +37,14 @@
     } else {
 	die "cannot fork: $!" unless defined($pid);
 	alarm $timeout;
-	exec ("../tools/lou_checktable $table --quiet");
+
+	# ignore tables with unicode-range: ucs4
+	if (system("grep -q '#+unicode-range: ucs4' $table") == 0) {
```

Comment:
You might want to make the space after the colon optional, and there could in theory also be one or more spaces before the colon.